.github/workflows/coverage.yml

```diff
@@ -1,20 +1,23 @@
-name: nim-libp2p codecov builds
+name: Coverage
 
 on:
-  #On push to common branches, this computes the "bases stats" for PRs
+  # On push to common branches, this computes the coverage PRs will compared against
   push:
     branches:
       - master
-      - unstable
   pull_request:
+    branches:
```

Comment:
I believe it should be as before, i.e., run for every PR.

Reply:
I'd suggest not taking the PR too seriously yet. It's on `Draft` and I haven't assigned reviewers yet for a reason :)

In any case, answering your question: The reason is there's a limit on the amount of codecov uploads with a free account. Maybe I'm getting too ahead of the problem, but we can discuss that later, when I finish making everything work :)